typescript/infra/config/environments/mainnet3/balances/desiredRebalancerBalances.json

```diff
@@ -0,0 +1,8 @@
+{
+  "arbitrum": 0.6,
+  "base": 0.2,
+  "ethereum": 0.3,
+  "optimism": 0.3,
+  "polygon": 130,
+  "unichain": 0.07
+}
```

Comment:
should we maybe have a test that ensure that all the chains in the mainnet cctp warp route have a desired balance setting here?  I think this can be something that can be easily forgotten to add if we cover a new chain that is in there but not here